pkg/plugins/policies/meshloadbalancingstrategy/plugin/v1alpha1/priority.go

```diff
@@ -0,0 +1,100 @@
+package v1alpha1
+
+import (
+	"math"
+
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
+	api "github.com/kumahq/kuma/pkg/plugins/policies/meshloadbalancingstrategy/api/v1alpha1"
+	"github.com/kumahq/kuma/pkg/util/pointer"
+)
+
+type LocalLbGroup struct {
+	Key    string
+	Value  string
+	Weight uint32
+}
+
+// Structure which hold information about cross zone with zones in the map to gice better access
+type CrossZoneLbGroup struct {
+	Type     api.ToZoneType
+	Zones    map[string]bool
+	Priority uint32
+}
+
+func GetPriority(conf *api.Conf, inboundTags mesh_proto.MultiValueTagSet, localZone string) ([]LocalLbGroup, []CrossZoneLbGroup) {
```

Comment:
it's not just priority but also weights for local zones. Should this be something like `GetLocalityGroups`?